reconstruction/MRI_reconstruction/varnet_demo/train.py

```diff
@@ -166,6 +163,9 @@ def trainer(args):
             final_shape = target.shape[-2:]
             max_value = torch.tensor(max_value).unsqueeze(0).to(device)
 
+            # create the loss function with data_range
+            loss_function = SSIMLoss(spatial_dims=2, data_range=max_value).to(device)
```

Comment:
could you please help verify this? @binliunls it seems this demo requires the large dataset...

Reply:
Hi @wyli ,
     Thank you for the quick fixing of this problem. I just verified this update and it works fine so far (the whole training task takes some time). I think this PR already fixed the issue in this [ticket](https://github.com/Project-MONAI/tutorials/issues/1349), in which the error happended at the beginning of the training process. 

BTW, just a naive question, should we set the `data_range` as a property that could be set and reset anytime, so that we don't need to initialize a new class every time we need to change the `data_range` parameter?

Thanks,
Bin

Reply:
sure, we can add a `set_data_range` function in the next version...I've updated the PR to use `loss_function.ssim_metric.data_range = max_value` for now, please review it again

Reply:
That works fine too. There is no question from my side. Thank you very much.